kolibri/logger/models.py

```diff
@@ -128,9 +132,23 @@ class UserSessionLog(BaseLogModel):
     user = models.ForeignKey(FacilityUser)
     channels = models.TextField(blank=True)
     start_timestamp = models.DateTimeField(auto_now_add=True)
-    completion_timestamp = models.DateTimeField(blank=True, null=True)
+    last_interaction_timestamp = models.DateTimeField(auto_now=True, null=True)
     pages = models.TextField(blank=True)
 
+    @classmethod
```

Comment:
Correct, this is MVP just to get time data recorded. We probably want to think carefully about exactly what data we want to track and how beyond that.